src/ztc/api/views/zaken.py

```diff
@@ -143,7 +144,10 @@ def perform_create(self, serializer):
                     msg = _(
                         f"Relations to non-concept {field_name} object can't be created"
                     )
-                    raise PermissionDenied(detail=msg)
+                    raise ValidationError(
```

Comment:
The `perform_create`, `perform_update` and `perform_destroy` code can be removed, see my comment here https://github.com/open-zaak/open-zaak/pull/137#discussion_r339043893

Reply:
I've removed the code, thanks for the remark!